src/renderers/WebGLCubeRenderTarget.js

```diff
@@ -103,12 +100,20 @@ WebGLCubeRenderTarget.prototype.fromEquirectangularTexture = function ( renderer
 
 	material.uniforms.tEquirect.value = texture;
 
-	const mesh = new Mesh( new BoxBufferGeometry( 5, 5, 5 ), material );
+	const mesh = new Mesh( geometry, material );
+
+	const currentMinFilter = texture.minFilter;
+	const currentRenderList = renderer.getRenderList();
+	const currentRenderTarget = renderer.getRenderTarget();
 
-	scene.add( mesh );
+	texture.minFilter = LinearFilter;
```

Comment:
I am concerned about this. The texture can be RGBE encoded, in which case linear filtering is inappropriate. See https://github.com/mrdoob/three.js/issues/19716#issuecomment-648242588.

Reply:
Should I change it to this?

```js
texture.minFilter = texture.encoding === RGBEEncoded ? NearestFilter : LinearFilter;
```

Reply:
If linear filtering is appropriate for a given encoding, then the texture loader sets it as linear. There should be no reason to reset it -- at least for HDR/EXR.

PNG files can be RGBM16 encoded. It is the user's responsibility to set these properties in that case, since the loader won't.

Why are you using a filter different from the one the user specifies? Can you explain what you want to achieve here? Disabling mips?

Reply:
> Can you explain what you want to achieve here? Disabling mips?

Yes.

Reply:
Please do not remove this code unless you can demonstrate a compelling reason to do so.
```js
this.texture.generateMipmaps = texture.generateMipmaps;
this.texture.minFilter = texture.minFilter;
this.texture.magFilter = texture.magFilter;
```

Remember, not all equirectangular maps are JPGs.

If you want to create a special case to accommodate the default settings of `TextureLoader`, that is up to you.

Reply:
I think I'm not doing a good job at explaining the reason of `texture.minFilter = LinearFilter`

If `texture.minFilter` is set to `LinearMipmapLinearFilter` then we end up with blurred poles like here:

<img width="500" alt="Screen Shot 2020-07-27 at 7 59 10 PM" src="https://user-images.githubusercontent.com/97088/88575430-d10d7c00-d07e-11ea-902b-75563a5a02a0.png">

By forcing `texture.minFilter = LinearFilter` before the render, we avoid mipmapping and we get crips poles:

<img width="497" alt="Screen Shot 2020-07-27 at 7 58 51 PM" src="https://user-images.githubusercontent.com/97088/88575452-d965b700-d07e-11ea-8784-b1e81bd7c4fa.png">

Makes sense?

Reply:
> Please do not remove this code unless you can demonstrate a compelling reason to do so.
```js
this.texture.generateMipmaps = texture.generateMipmaps;
this.texture.minFilter = texture.minFilter;
this.texture.magFilter = texture.magFilter;
```

I agree with that, but that's not what we're talking about here.

Reply:
The question is whether I should use this code before rendering the cubemap:

```js
texture.minFilter = texture.encoding === RGBEEncoded ? NearestFilter : LinearFilter;
```

Reply:
I do not see a difference in those pics, so I can't comment on that.
Have you tested other jpg textures -- both large and small -- to confirm this is an issue?
Does the issue not happen when the equirectangular texture is used directly as a skybox background or as an env map?

//

Maybe write it like this:
```js
if ( texture.minFilter === LinearMipmapLinearFilter ) texture.minFilter = LinearFilter;
```
//

Why should the min filter for the texture when rendering to the cube map be different from the min filter for the cube map when rendering to the scene?

Reply:
This is how the pole looks like without `texture.minFilter = LinearFilter`:

<img width="381" alt="Screen Shot 2020-08-02 at 12 18 12 AM" src="https://user-images.githubusercontent.com/97088/89111407-584c5c80-d490-11ea-81b6-94ec034cafd6.png">

This is how the pole looks like with `texture.minFilter = LinearFilter`:

<img width="369" alt="Screen Shot 2020-08-02 at 12 20 39 AM" src="https://user-images.githubusercontent.com/97088/89111435-ad886e00-d490-11ea-923a-20db92437c3b.png">

Reply:
Thanks for the pics.

If you want to make a special case, that is up to you. I just can't support doing so since I do not have an explanation as to why mip maps are suddenly not a good thing.

Maybe if you were able to answer my questions above, that would shed some light on it.

---

Aside:

HDR textures are now working for me, so this is the only remaining issue as far as I am concerned.

I think squashing the commits would make this PR easier to follow. I do not care about my original PR, which this extended.

Reply:
> If you want to make a special case, that is up to you. I just can't support doing so since I do not have an explanation as to why mip maps are suddenly not a good thing.

I think it's the same reason we were get a seam when mapping a equirectangular texture in a sphere. When I first bumped with that issue I remember asking on Twitter about it and Alex Evans answered this:

<img width="595" alt="Screen Shot 2020-08-02 at 9 54 25 AM" src="https://user-images.githubusercontent.com/97088/89118445-dc2e3500-d4e0-11ea-8cce-2bc680942fd0.png">

So in the case of a Sphere, because the GPU is dealing with doing 2x2 blocks, it finds a block that goes from UV.x 1.0 to 0.0 so it picks the smallest mip. That results in a seam when the UV wraps.

We don't have seams in this case because the shader in `fromEquirectangularTexture` doesn't use UVs.

However,  when the fragment shader is drawing the poles, because the 2x2 block approach, the further apart the pixels sampled are in the block the smallest the mip is, so we end up with blurred poles.

By disabling mipmapping we force the GPU skip that step.

For more research on the subject, I think this is the best query:
https://www.google.com/search?q=graphics+cards+2x2+pixel+blocks

Do you think my suspicion makes sense?

Reply:
Sorry. I just don't know at this point, unfortunately...

Reply:
Sorry, not sure I understand.

Do you mean that you don't know if doing this is right?

```js
texture.minFilter = texture.encoding === RGBEEncoded ? NearestFilter : LinearFilter;
```

Or that I was unsuccessful at explaining the issue and I confused you?

Reply:
I would write it like this:

>if ( texture.minFilter === LinearMipmapLinearFilter ) texture.minFilter = LinearFilter;

I feel uncomfortable about rendering with different settings than the user set, and I have not seen more than one texture that has this problem, so I would be reluctant to modify the library to accommodate it. 

However, if you want a special case, then that is fine with me. After all, you may be right.

Reply:
> I would write it like this:
>
>> if ( texture.minFilter === LinearMipmapLinearFilter ) texture.minFilter = LinearFilter;

That sounds good to me! 👍